pkg/autoscaler/autoscaler_test.go

```diff
@@ -291,31 +291,27 @@ func (r *mockReporter) Report(m Measurement, v float64) error {
 	return nil
 }
 
-func newTestAutoscaler(model v1alpha1.RevisionRequestConcurrencyModelType, targetConcurrency float64) *Autoscaler {
+func newTestAutoscaler(containerConcurrency int) *Autoscaler {
```

Comment:
We seem to be passing in `10.0` to this int in all the tests above -- should this be `10`?

Reply:
There is one test over a longer period of time that uses `1.0`.